app/views/admin_mailer/feedback.html.erb

```diff
@@ -1,15 +1,27 @@
 <% content_for :message do %>
 
-  <p><%= style_bold("Here's some user feedback on the archive!") %></p>
-  
+  <p><%= style_bold("Here's some user feedback on the Archive!") %></p>
+  <% if @username.blank? %>
```

Comment:
Could we shorten this a bit so it doesn't repeat as much, like
```
<p>Username: <%= @username.present? ? (raw @username) : "Anonymous user" %></p>
```
(Did I mangle the syntax? Probably!)